frontend/src/components/LandingPage.vue

```diff
@@ -397,8 +397,6 @@ export default {
   data() {
     return {
       input: '',
-      PATHS: PATHS,
-      results: {},
       showCancelDialog: false,
       ccofNewApplicationText: [
```

Comment:
While we're cleaning up data() can we move this into created()? It's just static content.